struct_test.go

```diff
@@ -364,6 +387,18 @@ names=alice, bruce`))
 	})
 }
 
+func Test_MapToExtended(t *testing.T) {
```

Comment:
The body of the test case seems identical to the one above, should we only keep one?

Reply:
Yes sorry this sneaked as I was testing this code whilst I was making changes to this code.

Reply:
No worries!